packages/flutter_tools/lib/src/commands/run_mojo.dart

```diff
@@ -58,6 +59,18 @@ class RunMojoCommand extends Command {
     return _makePathAbsolute(path.join(argResults['mojo-path'], 'out', mojoBuildType, 'mojo_shell'));
   }
 
+  BuildConfig _getCurrentHostConfig() {
+    BuildConfiguration result;
+    TargetPlatform target = getCurrentHostPlatformAsTarget();
+    for (BuildConfiguration config in buildConfigurations) {
+      if (config.targetPlatform == target) {
+        result = config;
+        break;
+      }
+    }
+    return result;
+  }
```

Comment:
It's not duplicated as far as I know. I could put it somewhere if it seems useful.